src/Data/Fin/Permutation/Transposition/List.agda

```diff
@@ -29,43 +31,68 @@ private
 
 -- This decomposition is not a unique representation of the original
 -- permutation but can be used to simply proofs about permutations (by
--- instead inducting on the list of transpositions).
+-- instead inducting on the list of transpositions, where a
+-- transposition is a permutation swapping two distinct elements and
+-- leaving the rest in place).
 
 TranspositionList : ℕ → Set
-TranspositionList n = List (Fin n × Fin n)
+TranspositionList n = List (Σ[ i ∈ Fin n ] ∃[ j ] i ≢ j)
```

Comment:
Maybe `List (∃₂ λ i j → i ≢ j)`? Could also write `(∃₂ _≢_)` but I'm not sure how clear that is?

Reply:
You need a hint that they have type `Fin n` rather than some other type, sadly.

Reply:
`List (∃₂ λ (i j : Fin n) → i ≢ j)` then?